core-plugin/src/com/google/gct/idea/debugger/CloudDebugProcess.java

```diff
@@ -501,8 +500,20 @@ public SaveAndExitAction() {
 
     @Override
     public void actionPerformed(AnActionEvent event) {
-      ExitDialog exitDialog = new ExitDialog(getXDebugSession().getProject());
-      exitDialog.showAndGetOk();
+      int result = Messages.showDialog(getXDebugSession().getProject(),
+          GctBundle.getString("clouddebug.continuelistening"),
+          GctBundle.getString("clouddebug.message.title"), new String[]{"Yes", "No"}, 1,
```

Comment:
UI is clearer when button names are verbal: e.g. "Continue listening", "Stop listening" since the user doesn't have to carefully parse the question to see which way it's phrased (Do continue listening, Don't continue listening?) I.e. depending on how you phrase the text, yes and no can mean one way or the other.